CMakeLists.txt

```diff
@@ -392,7 +392,8 @@ if(TOP_LEVEL_PROJECT)
                 "%define _use_internal_dependency_generator 0
                 %define __getdeps()while read file; do /usr/lib/rpm/rpmdeps -%{1} ${file} | %{__grep} -v libantlr3c.so | %{__grep} -v libtbb.so ; done | /bin/sort -u
                 %define __find_provides /bin/sh -c '%{__getdeps P}'
-                %define __find_requires /bin/sh -c '%{__getdeps R}'")
+                %define __find_requires /bin/sh -c '%{__getdeps R}
```

Comment:
Should there be ' on the end of this line?

Reply:
Fixed